### PR TITLE
frame_id string can be set as ros paramaters

### PR DESCRIPTION
**Pull request by marco-tranzatto**
frame_id was hard-coded to "world" in header section of output messages. 
Now it can be set as server parameter, and if it's not specified then "world" is used as default value.

@marija-p could you check this please?

### CODE REVIEW

**Comment by marija-p** [comment]
LGTM @marco-tranzatto . You can go ahead and merge. Thanks a lot again! :+1:

**Comment by marco-tranzatto** [comment]
@marija-p Thanks!
I cannot merge the pull request since I don't have write access. Would you mind doing it?

**Review by raghavkhanna — APPROVED** [review]

**Comment by marija-p** [comment]
Thanks @raghavkhanna !

**Pull request merged**
